lms/djangoapps/instructor/views/api_urls.py

```diff
@@ -4,7 +4,7 @@
 
 from django.conf.urls import url
 
-from lms.djangoapps.instructor.views import api, gradebook_api, writable_gradebook_api
```

Comment:
Where is the removal of this function?

Reply:
`writable_gradebook_api.py` is a file under `instructor.views`, and that whole file has been removed here.

Reply:
Ah... My bad. You are good here then.